docs/source/markdown/podman-image-scan.1.md

```diff
@@ -0,0 +1,43 @@
+% podman-image-scan(1)
+
+## NAME
+podman\-image\-scan - Scan an image's root filesystem for known vulnerabilities
+
+## SYNOPSIS
+**podman image scan** [*options*] [*image* ...] [*scanner options*]
+
+## DESCRIPTION
+Mounts the specified images' root file system and scans the contents accessible from
+the mount point for known vulnerabilities.
+
+## RETURN VALUE
+The scanner tool report.
+
+## OPTIONS
+
+#### **--scanner**, **-s** [*image*]
+
+A container image with the scanner tool installed and configured as the entrypoint and 
+reads the PODMAN_SCAN_MOUNT environment variable to determine where to scan.
```

Comment:
I prefer `PODMAN_SCAN_MOUNT` to be another flag/option on the CLI (e.g., `--mount-point`).  Mixing inputs on CLI flags and env variables can easily get confusing.